static/docs/user-guide/contributing-documentation.md

```diff
@@ -82,7 +82,7 @@ setting up.
 
 - Visit `http://localhost:3000/` and navigate to the docs in question.
 
-## Code style guidelines
+## Doc style guidelines (for Markdown)
```

Comment:
it's not only Markdown. It's JS as well.

Reply:
Also fixed in b866d1b